src/Compilers/VisualBasic/Test/Syntax/Scanner/ScannerTests.vb

```diff
@@ -508,9 +508,39 @@ Public Class ScannerTests
         Assert.Equal(" _ ", tk.ToFullString())
 
         tk = ScanOnce(" _'", startStatement:=True)
-        Assert.Equal(SyntaxKind.BadToken, tk.Kind)
+        Assert.Equal(SyntaxKind.EndOfFileToken, tk.Kind)
         Assert.Equal(" _'", tk.ToFullString())
-        Assert.Equal(30999, tk.Errors.First().Code)
+        Assert.True(tk.LeadingTrivia(0).Kind = SyntaxKind.WhitespaceTrivia)
+        Assert.True(tk.LeadingTrivia(1).Kind = SyntaxKind.LineContinuationTrivia)
+        Assert.True(tk.LeadingTrivia(2).Kind = SyntaxKind.CommentTrivia)
+        Assert.Equal(37306, tk.Errors.First().Code)
```

Comment:
>37306 [](start = 21, length = 5)

Please use ERRID.ERR_CommentsAfterLineContinuationNotAvailable1 instead of a literal, here and in other similar places

Reply:
The numbers were in the original code. I will change.

Reply:
> The numbers were in the original code. I will change.

Numbers for old errors won't change. However, it is very likely that we will have to adjust the number for the new error. Referring to the enum value is more robust and clearer.

---
In reply to: [217804005](https://github.com/dotnet/roslyn/pull/28327#discussion_r217804005) [](ancestors = 217804005)

Reply:
I can fix that for scanner tests, parser tests uses XMLLiterals so I added it to a PROTOTYPE comment and an Assert.